### ISSUE TITLE
Incorrect values on gravitational stabilizer?

### ISSUE THREAD

**Issue by Typheus** [simeonradivoev/MatterOverdrive#494]
Brief explanation of problem: Ok I'm going to go out on a limb and assume that the gravitational stabilizers have incorrect values on them for the break strength of the anomaly considering the anomaly already ate tritanium smooth plate right out from the hull and I had to switch to Tritanium Plate, and the strength isn't even 1, yet.

Steps to reproduce: Just look at any gravitational stabilizer

Crash report paste (if any) on [gist](gist.github.com):

MatterOverdrive version: 0.4.1

Minecraft version: 1.7.10

Screenshot provided: 
![2016-05-02_18 16 05](https://cloud.githubusercontent.com/assets/10276357/14969894/d8389ad8-1092-11e6-9f75-b0ddc5c98a07.png)

**Comment by simeonradivoev** [comment]
The values should be correct. The smooth tritanium plate may just have a low hardness.
The anomaly also breaks any blocks except bedrock, when they are in it's event horizon.

**Comment by Typheus** [comment]
I did an experiment with some dirt blocks since they have a value of 0.5. I figured if they didn't break, then it was absolutely correct.

I figured out what was going wrong that I didn't notice before. It wasn't breaking the blocks. It was pulling them towards it so that they would actually migrate around and some of the floor fell out for this reason.

Thanks for the quick response regarding the issue.

This can be considered closed, if you want.
